contracts/truefi2/PoolFactory.sol

```diff
@@ -171,6 +175,7 @@ contract PoolFactory is IPoolFactory, UpgradeableClaimable {
         ftlAgency = _ftlAgency;
         safu = _safu;
         loanFactory = _loanFactory;
+        borrowingMutex = _borrowingMutex;
```

Comment:
We'll also need to add a setter + tests for borrowingMutex since this is an existing contract. Let's do that in a follow-up PR.

Reply:
I actually think we should add it in this PR